src/main/java/org/shanerx/tradeshop/listeners/CustomInventoryListener.java

```diff
@@ -27,14 +27,15 @@
 
 import org.bukkit.ChatColor;
 import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
 import org.bukkit.event.Listener;
 import org.bukkit.event.inventory.InventoryClickEvent;
 import org.bukkit.inventory.Inventory;
 import org.shanerx.tradeshop.utils.Utils;
 
 public class CustomInventoryListener extends Utils implements Listener {
 
-	@EventHandler
+    @EventHandler(priority = EventPriority.HIGHEST)
```

Comment:
I know the `ignoreCancelled` flag does not seem to be working, but did you not put it here on purpose?

Reply:
yes because i do want to check it no matter what since its checking for our own inventory

Reply:
I guess, but what if some other plugin has anything to say about it?